### PR TITLE
[IMP] mail: introduce ChannelMemberListCategoryView/members

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2864112

### CODE REVIEW

File: addons/mail/static/src/models/channel_member_list_category_view.js [hunk 1]
```diff
@@ -1,12 +1,47 @@
...
+        _computeMembers() {
+            if (!this.exists()) {
+                return clear();
+            }
+            if (this.channelMemberListViewOwnerAsOnline) {
+                return replace(this.channelMemberListViewOwnerAsOnline.channel.orderedOnlineMembers);
+            }
+            if (this.channelMemberListViewOwnerAsOffline) {
+                return replace(this.channelMemberListViewOwnerAsOffline.channel.orderedOfflineMembers);
+            }
```
Comment (phenix-factory) [inline addons/mail/static/src/models/channel_member_list_category_view.js:36]: You can now use this.channel directly.

Comment (alexkuhn) [reply addons/mail/static/src/models/channel_member_list_category_view.js]: Changes done

**Review by phenix-factory — APPROVED** [review]
LGTM

**Comment by alexkuhn** [comment]
Moved to grouped PR: https://github.com/odoo/odoo/pull/93330

**Pull request closed without merging**